examples/matlab_mex/linear_mass_spring_model/linear_mass_spring_model.m

```diff
@@ -1,7 +1,5 @@
 function model = linear_mass_spring_model()
 
-import casadi.*
```

Comment:
why removing this?

Reply:
i reverted the change, this should not have gone into this pull request (was related to my octave issue). But generally i would prefer not using * imports.